datetime_tz/__init__.py

```diff
@@ -299,9 +410,40 @@ def _detect_timezone_php():
   if len(matches) > 1:
     warnings.warn("We detected multiple matches for the timezone, choosing "
                   "the first %s. (Matches where %s)" % (matches[0], matches))
+  if matches:
     return pytz.timezone(matches[0])
 
 
+class _default_tzinfos(object):
```

Comment:
No - it's used as in the tests.  If you want to use dateutil.parse with timezone support, but want it to return pytz timezones instead of regular Python tzinfos, you pass in this object as the 'tzinfos' argument.